template/glotzerlab-software.jinja

```diff
@@ -4,6 +4,11 @@
 {% set RUN = 'RUN' %}
 {% endif %}
 
+{{ RUN }} git clone --recursive --branch {{ COXETER_VERSION }} --depth 1 https://github.com/glotzerlab/coxeter \
+    && python3 -m pytest ./coxeter \
```

Comment:
This is failing with a message about doctest fixtures: https://dev.azure.com/glotzerlab/software/_build/results?buildId=6531&view=logs&j=c575ca02-ce81-55f6-3641-c29e1e00e512&t=7fc1a229-3e4e-569d-7dc5-9d02e6e8f0d3&l=9404

I'm guessing this is because coxeter has not yet been installed. Try moving this RUN block to `test.jinja` so it will run after all the software is installed.

Also, I'd prev